config/install/core.base_field_override.node.title.promote.yml

```diff
@@ -0,0 +1,21 @@
+langcode: en
```

Comment:
This file can be removed. It refers to the node type "title" which we need to remove in favor of "people".